thunder/torch/__init__.py

```diff
@@ -478,12 +478,6 @@ def multinomial(
         generator is None, lambda: f"multinomial does not yet support specifying a generator", NotImplementedError
     )
 
-    utils.check(
```

Comment:
redundant - above check verifies the same condition.

Reply:
Nice fix